chingu-voyages/v43-tier1-team-08
# 50 ticket bad words filtering ec

## Description

I modified the App.JS file to include a filter that will scan the user input for profanity and replace them with asterisks.  I am still trying to figure out how to take the input from the filter and back into the words[] array.
## Related Issue

#50 

## Acceptance Criteria

This branch should satisfy the filtering of profane words using the npm package "bad-words"

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates
Only the App.JS file has been updated and also I noted the dependency on the README.md section.

## Testing Steps / QA Criteria

Simply run the app on your local machine and make sure you include a profane word.
